modules/bridge/src/main/ts/ephox/bridge/api/Registry.ts

```diff
@@ -32,6 +32,7 @@ export interface Registry {
   addAutocompleter: (name: string, spec: AutocompleterSpec) => void;
   addSidebar: (name: string, spec: SidebarSpec) => void;
   addView: (name: string, spec: ViewSpec) => void;
+  addContext: (name: string, pred: (...args: any[]) => boolean) => void;
```

Comment:
Wouldn't this be `string[]` instead of any since it's something we produce? Also if that is the case then we should change all the other occurrences of any. I think we should try to avoid the `any` type if we can.

Reply:
The first argument would be the Editor, but since this is in bridge. It doesn't have the type for Editor since it only deals with `specs` 😞 I was thinking of (editor: Editor, args: string[])

Reply:
Hmm, do we need the editor since the registry is on the editor you would have that in scope in the closure anyway. We should log a tech dept to get tid of bridge.

Reply:
Right 🤦‍♂️ Pushed a commit to fix that.